### DISCUSSION TITLE
电犀牛 R68S ImmortalWrt 刷机 & 救砖说明

### DISCUSSION THREAD

**Discussion by 1715173329** [1715173329/blog#14] [Comments]
https://blog.imouto.in/post/openwrt/2022/flash-immortalwrt-for-fastrhino-r68s/

**Comment by Lixuhuilll** [comment]
感谢教程，已成功刷入使用，顺便补充一下，ImmortalWrt默认是没有luci界面的，需要ssh连接后安装，ssh默认没有密码，地址默认192.168.1.1
`opkg update && opkg install luci && opkg install luci-i18n-base-zh-cn`

**Comment by zqy7788789** [comment]
真复杂

**Comment by menphy1989** [comment]
请问提示下载Boot失败该如何解决？
